NEWS.md

```diff
@@ -1,11 +1,15 @@
 # rlistings 0.1.1.9006
 
-## rlistings 0.1.1
- * add title, subtitle, and (main and prov) footer support
- * now depends on dplyr instea dof magrittr to hopefully avoid var_labels droppage issues
- * `paginate_listing` now supports pagination in both directions
+### Enhancements
+ * Improved source and test file organization.
+ * Polished `DESCRIPTION` file.
```

Comment:
Changes to dependencies should be listed here explictly, rather than grouped under "Polished DESCRIPTION file"

Also I'm not sure 'improved soruce and test file organization' needs to be in there. Users don't care about that at all.

Reply:
You are right. I summarized it a bit better now, let me know what you think.